### PR TITLE
Upgrade to Rails 6.1

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: config/environments/test.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  # The test environment is used exclusively to run your application's
-  # test suite. You never need to work with it otherwise. Remember that
-  # your test database is "scratch space" for the test suite and is wiped
-  # and recreated between test runs. Don't rely on the data there!
-  config.cache_classes = true
+  config.cache_classes = false
```
Comment (justinlittman) [inline config/environments/test.rb:9]: Was this switch to `false` intentional?

File: config/initializers/filter_parameter_logging.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
```
Comment (justinlittman) [inline config/initializers/filter_parameter_logging.rb:4]: `:password` was dropped. Intentional?

**Review by justinlittman — CHANGES_REQUESTED** [review]
Just a couple of questions and file conflict.

File: config/initializers/filter_parameter_logging.rb (hunk 2 above)
Comment (jcoyne) [reply config/initializers/filter_parameter_logging.rb:4]: yes, but we don't collect passwords, right?

File: config/environments/test.rb (hunk 1 above)
Comment (jcoyne) [reply config/environments/test.rb:9]: yes.  this is the Rails default caching doesn't benefit us in test mode.

**Review by justinlittman — APPROVED** [review]

**Pull request merged**